src/fsharp/FSharp.Core.Unittests/FSharp.Core/Microsoft.FSharp.Collections/StringModule.fs

```diff
@@ -41,8 +41,8 @@ type StringModule() =
         let e6 = String.concat "foo" ["bar"]
         Assert.AreEqual("bar", e6)
         
-        let e7 = String.concat "foo" ["bav";"baz"]
-        Assert.AreEqual("bavfoobaz", e7)
+        let e7 = String.concat "foo" ["bar";"baz"]
+        Assert.AreEqual("barfoobaz", e7)
```

Comment:
This particular one is about "barf", not foo+bar. IIRC, a surprisingly wide array of accidental bodily fluid references had to be removed... :sweat_drops: